go.mod

```diff
@@ -60,7 +58,6 @@ require (
 require (
 	github.com/blang/semver v3.5.1+incompatible
 	github.com/charmbracelet/lipgloss v0.9.1
-	github.com/dagger/dagger/internal/mage v0.0.0-00010101000000-000000000000
```

Comment:
This is essentially the whole idea behind the second commit here - we get to remove this *insane* version id, which doesn't actually resolve to anything.